interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -145,7 +145,7 @@
         } else if ($colname == 'DOB' || $colname == 'regdate' || $colname == 'ad_reviewed' || $colname == 'userdate1') {
             $arow[] = oeFormatShortDate($row[$colname]);
         } else {
-            $arow[] = $row[$colname];
+            $arow[] = xls($row[$colname]);
```

Comment:
I don't know he answer to this, but would it be better to instead escape this when do the json_encode()?(it looks like that function has some escaping options)

Reply:
Of course you right. this is important to escape all the chars in the json (we don't know how safe is Datatable..)
I add to json_encode options that contains all the spacial chars (like 'text' function ), bitmask of JSON_HEX_TAG (<)(>), JSON_HEX_AMP(&), JSON_HEX_APOS('), JSON_HEX_QUOT(") together.
Hopefully you agree with me.

Amiel

Reply:
Very nice. Agree and check out my comment below.